jpo-ode-common/src/main/java/us/dot/its/jpo/ode/inet/InetPacketException.java

```diff
@@ -1,29 +1,30 @@
 /*******************************************************************************
  * Copyright 2018 572682
- * 
+ *
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not
  * use this file except in compliance with the License.  You may obtain a copy
  * of the License at
- * 
+ *
  *   http://www.apache.org/licenses/LICENSE-2.0
- * 
+ *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
  * License for the specific language governing permissions and limitations under
  * the License.
  ******************************************************************************/
+
 package us.dot.its.jpo.ode.inet;
 
 public class InetPacketException extends Exception {
-	
-	private static final long serialVersionUID = 1L;
 
-	public InetPacketException(String message) {
-		super(message);
-	}
-	
-	public InetPacketException(String message, Throwable cause) {
-		super(message, cause);
-	}
+  private static final long serialVersionUID = 1L;
+
+  public InetPacketException(String message) {
+    super(message);
+  }
+
+  public InetPacketException(String message, Throwable cause) {
+     super(message, cause);
```

Comment:
This is an example of the warnings developers will receive should they forget or fail to abide by the checkstyle style guide. It won't force anyone to resolve style issues before merging (because that would be cumbersome) but it will be annoying and loud enough to get people to pay attention to style violations